helper/memory_calculator.go

```diff
@@ -167,3 +162,22 @@ func (m MemoryCalculator) Execute() (map[string]string, error) {
 
 	return map[string]string{"JAVA_TOOL_OPTIONS": strings.Join(values, " ")}, nil
 }
+
+func parseMemInfo(s string) (int64, error) {
+	pattern := `MemAvailable:\s*(\d+)(.*)`
```

Comment:
Can we always be sure that the amount is shown as kB?
My approach was that in case of a different unit it would be safer to not use it.

Reply:
https://github.com/paketo-buildpacks/libjvm/pull/46/files#diff-0ab20a99dcc0be1692ffca784255e81c58b78b03b66a6bfc65f77b6e82e3e16aR178-R182178 is now parsing the units explicitly. In practice the unit appears to always be `kB` but the [documentation](https://man7.org/linux/man-pages/man5/proc.5.html) potentially leaves open the option for other units so I decided to go with the more complete/paranoid version while I was making changes.

Reply:
sorry totally missed the second match group.